.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           cp .env.ci .env
           npm install
           node setup --no-auth --no-tls
+          rm .finish-setup
```

Comment:
this one threw me for a loop for a while: it would cause SQLite UNIQUE CONSTRAINT errors because the oauth test would consider the login attempt a "first time login attemp" and would try to elevate `Test User` to an admin. Except the test builder already creates a `test admin` and `test user`, and the latter has a slug-conflict with `Test User`.